app/stores/ada/DaedalusTransferStore.js

```diff
@@ -110,19 +110,23 @@ export default class DaedalusTransferStore extends Store {
     wallet: RustModule.Wallet.DaedalusWallet,
   ): void => {
     this._updateStatus('restoringAddresses');
-    this.ws = new WebSocket(websocketUrl);
-    this.ws.addEventListener('open', () => {
+    runInAction(() => {
```

Comment:
Why wrapping up `this.ws` inside a `runInAction` if this was already only called by an action?

Reply:
whenever you have an async call, the continuation is no longer an `action` so you have to either re-add `@action` or use `runInAction` again